### PR TITLE
Added example notebooks from PyData Carolinas talk.

### PR DESCRIPTION
**Pull request by mariusvniekerk**
Initial example notebooks.

### CODE REVIEW

File: examples/readme.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+A list of example notebooks for how to use some features of spylon.
```
Comment (parente) [inline examples/readme.rst]: Should there be links to the notebooks here on nbviewer or github?

File: examples/01 SpylonExample.ipynb [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+   ],
+   "source": [
+    "rand"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# You "
```
Comment (parente) [inline examples/01 SpylonExample.ipynb]: Me?

Comment (mariusvniekerk) [inline examples/01 SpylonExample.ipynb]: Fixed.

File: examples/readme.rst (hunk 1 above)
Comment (mariusvniekerk) [inline examples/readme.rst]: Fixed.

**Pull request merged**